packages/ag-charts-community/src/api/agCharts.ts

```diff
@@ -296,7 +258,7 @@ class AgChartsInternal {
             false
         );
 
-        debugOptions('>>> AgCharts.updateUserDelta() user delta', deltaOptions);
+        debug(() => ['>>> AgCharts.updateUserDelta() user delta', deltaOptions]);
```

Comment:
Nit: Re-add `deepClone()` here and other cases where we were calling `debugOptions()`?